### PR TITLE
Update controllers adding dt in to update as in ros2_control

### PR DESCRIPTION
**Pull request by destogl**
This PR is related/depends on merge on ros-controls/ros2_control#520 (and ros2_control/ros2_controllers#241)﻿

### CODE REVIEW

File: ur_controllers/src/speed_scaling_state_broadcaster.cpp [hunk 1]
```diff
@@ -93,8 +93,7 @@ SpeedScalingStateBroadcaster::on_configure(const rclcpp_lifecycle::State& /*prev
 rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
 SpeedScalingStateBroadcaster::on_activate(const rclcpp_lifecycle::State& /*previous_state*/)
 {
-  last_publish_time_ = node_->now();
-  return rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn::SUCCESS;
+  //   return rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn::SUCCESS;
```
Comment (livanov93) [inline ur_controllers/src/speed_scaling_state_broadcaster.cpp:96]: No return value?

**Review by livanov93 — CHANGES_REQUESTED** [review]
Just small change

File: ur_controllers/src/speed_scaling_state_broadcaster.cpp (hunk 1 above)
Comment (destogl) [reply ur_controllers/src/speed_scaling_state_broadcaster.cpp:96]: Thanks for catching this!

Comment (destogl) [reply ur_controllers/src/speed_scaling_state_broadcaster.cpp:96]:
```suggestion
  return rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn::SUCCESS;
```

**Comment by destogl** [comment]
@livanov93 this seems to be working. Can you give it a try with testing?

**Comment by livanov93** [comment]
@destogl It looks good. I can test it with ursim immediately. Hardware test will have to wait.

**Review by livanov93 — APPROVED** [review]
@destogl Tested with ursim on galactic. Works normally.

**Pull request merged**